### ISSUE TITLE
Fail to revoke cfssl certification

### ISSUE THREAD

**Issue by abawchen** [Netflix/lemur#2836]
Not sure it supports cfssl revocation.

And it seems the schema of table `certificates` created by `lemur init` is not compatible with [certdb of cfssl](https://github.com/cloudflare/cfssl/tree/master/certdb), because when I use curl command to send data from `lemur` directly:

```
curl -d '{"serial": "549743092010436041874636640786015368771950028642","authority_key_id": "26896e8bbe6e6af00f3436712e5bc4b7d7eb1dda", "reason": "superseded"}' http://192.168.0.168:8888/api/v1/cfssl/revoke
```

it shows:
```
{"success":false,"result":null,"errors":[{"code":11000,"message":"pq: column \"serial_number\" does not exist"}],"messages":[]}
```

**Comment by geekspeed** [comment]
Looks like you are trying to hit cfssl not lemur

**Comment by rohanmehto2** [comment]
@abawchen Were you able to have a working implementation with cfssl revocation functional?

**Comment by abawchen** [comment]
@rohanmehto2 : Yes, I customize the revoke function by myself, but not an elegant one -- some hardcode and it only works with cfssl.

Do you need the code snippet?

**Comment by rohanmehto2** [comment]
@abawchen It would be really great if you could share the snippet. I'm also using cfssl only so there shouldn't be any problem. Thanks
